Grid.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Grid</title>
+    <link href="https://fonts.googleapis.com/css2?family=Nunito&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="/ComponentLibrary/Css/main.css">
+
```

Comment:
I guess you have not linked your css file with grids
![image](https://user-images.githubusercontent.com/55906931/153596104-a8942243-2d94-495e-ba7b-a2991e908ddb.png)

Reply:
Yeah saksham will resolve that thanks for reviewing and pointing out.